com.nickgirga.webready.yml

```diff
@@ -0,0 +1,37 @@
+id: com.nickgirga.webready
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for easy access to videos and "Open With"
+  - --filesystem=home
```

Comment:
Seems to use the portal file. Doesn't seem to be required.

Reply:
The file chooser works okay without that permission, but the output file (which should end up right next to the source video file) will end up in the "/run/user/..." directories instead, which is not desirable behavior.

Reply:
Ah right forgot about that